cli/utilities/containers.py

```diff
@@ -113,17 +121,33 @@ def run(
         if detach:
             cmd += " -d"
 
+        if interactive:
+            cmd += " -i"
+
+        if tty:
+            cmd += " -t"
+
         if rm:
-            cmd += f" --rm {rm}"
+            cmd += " --rm"
+
+        if user:
+            cmd += f" --user {user}"
+
+        if entry_point:
+            cmd += f" --entrypoint {entry_point}"
 
         if image:
             cmd += f" {image}"
 
         if cmds:
             cmd += f" {cmds}"
 
-        if self.execute(sudo=True, long_running=True, cmd=cmd):
+        out = self.execute(sudo=True, cmd=cmd)
```

Comment:
Revert this change, this should be long running command. It will affect existing test cases.